src/Microsoft.AspNet.Mvc.Extensions/Rendering/IHtmlHelper.cs

```diff
@@ -103,7 +103,7 @@ public interface IHtmlHelper
         /// &lt;form&gt; is submitted.
         /// </summary>
         /// <returns>A new <see cref="HtmlString"/> containing the &lt;hidden&gt; element.</returns>
-        HtmlString AntiForgeryToken();
+        HtmlString AntiforgeryToken();
```

Comment:
Yep. Will discuss with @Eilon and get a decision to either keep this or revert it.